pkgs/tools/typesetting/tex/texlive/combine.nix

```
@@ -225,14 +225,17 @@ in (buildEnv {
 
     perl `type -P mktexlsr.pl` --sort ./share/texmf
     ${bin.texlinks}/bin/texlinks "$out/bin" && wrapBin
-    perl `type -P fmtutil.pl` --sys --all | grep '^fmtutil' # too verbose
+    FORCE_SOURCE_DATE=1 perl `type -P fmtutil.pl` --sys --all | grep '^fmtutil' # too verbose
     #${bin.texlinks}/bin/texlinks "$out/bin" && wrapBin # do we need to regenerate format links?
 
     # Disable unavailable map files
     echo y | perl `type -P updmap.pl` --sys --syncwithtrees --force
     # Regenerate the map files (this is optional)
     perl `type -P updmap.pl` --sys --force
 
+    # sort entries to improve reproducibility
+    sort -o $out/share/texmf-config/web2c/updmap.cfg $out/share/texmf-config/web2c/updmap.cfg
```

```suggestion
    sort -o "$TEXMFVAR"/web2c/updmap.cfg "$TEXMFVAR"/web2c/updmap.cfg
```
we don't use `texmf-config`, actually.

---

I take it back: on my system, I only have `share/texmf/web2c/updmap.cfg`, which is read-only (this is with scheme-full). So you simply need to test whether the file was created.

---

Thank you! I only tested with `scheme-small` and a couple of packages. With the fix, `tests.texlive` pass.